homeassistant/components/sensor/verisure.py

```diff
@@ -98,3 +105,34 @@ def unit_of_measurement(self):
     def update(self):
         """ update sensor """
         verisure.update_climate()
+
+
+class VerisureMouseDetection(Entity):
+    """ represents a Verisure mousedetector within home assistant. """
+
+    def __init__(self, mousedetection_status):
+        self._id = mousedetection_status.deviceLabel
+
+    @property
+    def name(self):
+        """ Returns the name of the device. """
+        return '{} {}'.format(
+            verisure.MOUSEDETECTION_STATUS[self._id].location,
+            "Mouse")
+
+    @property
+    def state(self):
+        """ Returns the state of the device. """
+        return verisure.MOUSEDETECTION_STATUS[self._id].count
+
+    @property
+    def unit_of_measurement(self):
+        """ Unit of measurement of this entity """
+        if verisure.MOUSEDETECTION_STATUS[self._id].count >= 1:
```

Comment:
You don't want to change the unit based on the count. If you do so, code looking at the data will not be able to group the states